docs/intro.rst

```diff
@@ -101,6 +101,34 @@ You can now take advantage of the Tutor-powered CLI (item #3) to bootstrap your
 
 Under the hood, Tutor simply runs ``docker-compose`` and ``docker`` commands to launch your platform. These commands are printed in the standard output, such that you are free to replicate the same behaviour by simply copying/pasting the same commands.
 
+How do I navigate Tutor's command-line interface?
+-------------------------------------------------
```

Comment:
I guess this section should include a link to the CLI reference docs, somewhere.

Reply:
Added